core/src/components/datetime/datetime.tsx

```diff
@@ -326,7 +332,15 @@ export class Datetime implements ComponentInterface {
              * there can be 1 hr difference when dealing w/ DST
              */
             const date = new Date(convertDataToISO(this.datetimeValue));
-            this.datetimeValue.tzOffset = date.getTimezoneOffset() * -1;
+
+            // If a custom display timezone is provided, use that tzOffset value instead
+            if (typeof(this.displayTimezone) !== 'undefined' && this.displayTimezone.length > 0) {
+              const utcDateTime = new Date(date.toLocaleString('en-US', { timeZone: 'utc' }));
```

Comment:
This should be fine to default to `en-US` here right, since we're just manipulating these dates internally for now right?

Reply:
exactly, its simply used to determine the offsets only